CMakeLists.txt

```diff
@@ -136,10 +136,10 @@ if (${REST_MPFR} MATCHES "ON")
     find_path(MPFR_INCLUDE mpfr.h HINTS ${MPFR_PATH} ${MPFR_PATH}/include) # path to include directory
     find_path(MPFR_LIB_PATH libmpfr.so HINTS ${MPFR_PATH} ${MPFR_PATH}/lib)
     find_library(MPFR_LIB mpfr HINTS ${MPFR_PATH} ${MPFR_PATH}/lib) # path to .so file
+    add_compile_definitions(USE_MPFR)
```

Comment:
@lobis shouldn't be placed inside the second if condition?

Reply:
In my local copy it is inside the `if( NOT MPFR_INCLUDE ...` and it seems to be working properly

Reply:
If MPFR libraries are not found then code using MPFR libraries should not be exposed.

Reply:
but isn't this the original location? that is why I changed it... you moved it in the first place no?